src/doc/CreateHighlightDialog.js

```diff
@@ -63,10 +61,13 @@ class CreateHighlightDialog extends EventEmitter {
     }
 
     unmountPopover() {
-        if (this.createPopoverComponent && this.popoverLayerEl) {
-            unmountComponentAtNode(this.popoverLayerEl);
+        const pageEl = this.isMobile
```

Comment:
I've seen this code repeated throughout the code, maybe there's someway we can turn it into a generic util method that can be called across more classes besides `AnnotationThread.getPopoverParent()`? 

The `isMobile` check also feels like it should be a utility method instead of a instance variable to be more dynamic with page resizes

Reply:
moved into util methods